cfy_manager/components/restservice/restservice.py

```diff
@@ -355,20 +410,74 @@ def _upload_cloudify_license():
 
     def install(self):
         logger.notice('Installing Rest Service...')
-        self._install()
+        yum_install(config[RESTSERVICE][SOURCES]['restservice_source_url'])
+        yum_install(config[RESTSERVICE][SOURCES]['agents_source_url'])
+
+        self._chown_resources_dir()
+
+        set_logrotate(RESTSERVICE)
+
+        if DATABASE_SERVICE not in config[SERVICES_TO_INSTALL]:
+            yum_install(config[RESTSERVICE][SOURCES]['haproxy_rpm_url'])
         logger.notice('Rest Service successfully installed')
 
     def configure(self):
         logger.notice('Configuring Rest Service...')
-        self._configure()
+        if (
+            DATABASE_SERVICE not in config[SERVICES_TO_INSTALL]
+            and config[POSTGRESQL_CLIENT]['host'] in (
+                '127.0.0.1', 'localhost',
```

Comment:
this needs to actually be set in the config, right? as in, it's not being set anywhere by the install process?

Reply:
You mean the postgresql_client.host setting? We default it to localhost.